frontend/src/metabase/visualizations/lib/chart_values.js

```diff
@@ -263,11 +263,16 @@ export function onRenderValueLabels(
         // only create labels for the correct class(es) given the type of label
         .filter(d => !(d.rotated ^ (klass === "value-label-white")))
         .attr("class", klass)
-        .text(({ y, seriesIndex }) =>
-          formatYValue(y, {
+        .text(({ y, seriesIndex }) => {
+          const text = formatYValue(y, {
             compact: compact === null ? compactForSeries[seriesIndex] : compact,
-          }),
-        ),
+          });
+          return y < 0 &&
+            text[0] === "-" &&
+            displays[seriesIndex] === "waterfall"
+            ? "(" + text.substr(1) + ")"
```

Comment:
@paulrosenzweig @mazameli I'm sure this workaround isn't really acceptable.
Alas, the more correct solution (and more work) is to extend `frontend/src/metabase/lib/formatting.js` to have an option, e.g. `negativeInParentheses`. Do you think that it's the right path to go? Or is there a better alternative?

Reply:
I agree that extending `formatNumber` with that option is probably the best route. It's probably not much more code, and it becomes easier to unit test over there. Maybe something like this?
```js
if (value < 0 && settingIsTrue)  {
  return "(" +  formatNumber(-value, optionsWithoutSetting) + ")";
}
```

Separately, I hadn't realized that this is totally new formatting for us. I guess we'll want to add an option so people can use it elsewhere (in tables, probably). It seems kinda weird that this is the only place in the app you can show negatives in parens.